sdk/ai/azure-ai-projects/samples/connections/async_samples/sample_connections_async.py

```diff
@@ -23,15 +23,17 @@
        in your AI Studio Hub page.
     3) MODEL_DEPLOYMENT_NAME - The model deployment name, as found in your AI Studio Project.
 """
+from typing import cast
 
 import asyncio
 import os
 from azure.ai.projects.aio import AIProjectClient
 from azure.ai.projects.models import ConnectionType, AuthenticationType
+from azure.core.credentials_async import AsyncTokenCredential
 from azure.identity.aio import DefaultAzureCredential
```

Comment:
Please put the line `from azure.core.credentials_async import AsyncTokenCredential` above the line that uses it, not at the top.